src/Compilers/CSharp/Portable/Parser/LanguageParser.cs

```diff
@@ -11038,7 +11038,7 @@ private ExpressionSyntax ParseExpressionContinued(ExpressionSyntax unaryOrPrimar
                     return null;
 
                 // Now consume the operator (including consuming multiple tokens in the case of merged operator tokens)
-                var operatorToken = eatOperatorToken(operatorTokenKind);
+                var operatorToken = EatOperatorToken(operatorTokenKind);
```

Comment:
EatExpressionOperatorToken.  Absent that, it's less clear.  For example, is this for eating the token in an operator declaration?